build/tasks/gemini.js

```diff
@@ -20,7 +20,7 @@ var config = {
     }
 };
 
-gulp.task('css:reference', ["build"], function() {
+gulp.task('css:reference', ["build"], function(cb) {
```

Comment:
I'm assuming this is for callback, right? 
Isn't cb supposed to be called somewhere at the end  of this task? `cb()`?